VimeoNetworking/Sources/Models/Subscription.swift

```diff
@@ -70,4 +70,22 @@ public class Subscription: VIMModelObject
             "vod_rental_expiration_warning": "vodRentalExpirationWarning",
             "account_expiration_warning": "accountExpirationWarning"]
     }
+    
+    // MARK: - Helpers
+    
+    /// Helper method that determine whether a user has all the subscription settings turned off.
+    ///
+    /// - Returns: A boolean that indicates whether the user has all the settings for push notifications disabled.
+    public func areSubscriptionsDisabled() -> Bool
+    {
+        return (self.comment == false &&
+                self.credit == false &&
+                self.like == false &&
+                self.mention == false &&
+                self.reply == false &&
+                self.follow == false &&
+                self.vodPreorderAvailable == false &&
+                self.videoAvailable == false &&
+                self.share == false)
+    }
```

Comment:
^ That's a neat idea. 

As another alternative, it would be cool to have some type of data structure that maintains this list, even if it's just a simple array that's a computed var returning these in a list, that can be filtered to determine if any values are false.